### PR TITLE
SEOImages: User Story 1463880, Part 8

### PR DESCRIPTION
**Pull request by damabe**
## Summary

Updating image file names and references to improve SEO.

Fixes #1463880

Contributes to #10839

### CODE REVIEW

**Comment by damabe** [comment]
@mairaw This PR is ready for review. CATS report: https://contentqacats.azurewebsites.net/CheckResults/2019031200048/Report/.

File: docs/framework/wpf/advanced/bidirectional-features-in-wpf-overview.md [hunk 1]
```diff
@@ -57,17 +53,13 @@ Unlike any other development platform, [!INCLUDE[TLA2#tla_winclient](../../../..
   
  The first document example contains a document that has a number of network share names; for example `\\server1\folder\file.ext`. Whether you have this network link in an Arabic or English document, you always want it to appear in the same way. The following graphic shows the link in an Arabic <xref:System.Windows.FlowDirection.RightToLeft> document.
```
Comment (mairaw) [inline docs/framework/wpf/advanced/bidirectional-features-in-wpf-overview.md]:
```suggestion
 The first document example contains a document that has a number of network share names; for example `\\server1\folder\file.ext`. Whether you have this network link in an Arabic or English document, you always want it to appear in the same way. The following graphic illustrates using the Span element and shows the link in an Arabic <xref:System.Windows.FlowDirection.RightToLeft> document:
```

File: docs/framework/wpf/advanced/bidirectional-features-in-wpf-overview.md [hunk 2]
```diff
@@ -57,17 +53,13 @@ Unlike any other development platform, [!INCLUDE[TLA2#tla_winclient](../../../..
...
-  
- ![Document that flows from right to left](./media/flowdocument.PNG "FlowDocument")  
+ ![Graphic that illustrates using the Span element.](./media/bidirectional-features-in-wpf-overview/flow-direction-span-element.png "FlowDocument")  
   
  Because the text is <xref:System.Windows.FlowDirection.RightToLeft>, all special characters, such as the "\\", separate the text in a right to left order. That results in the link not being shown in the correct order, therefore to solve the problem, the text must be embedded to preserve a separate <xref:System.Windows.Documents.Run> flowing <xref:System.Windows.FlowDirection.LeftToRight>. Instead of having a separate <xref:System.Windows.Documents.Run> for each language, a better way to solve the problem is to embed the less frequently used English text into a larger Arabic <xref:System.Windows.Documents.Span>.  
   
- The following graphic illustrates this.  
-  
- **Graphic That Illustrates Using the Run Element Embedded in a Span Element**  
+ The following graphic illustrates this.
```
Comment (mairaw) [inline docs/framework/wpf/advanced/bidirectional-features-in-wpf-overview.md]:
```suggestion
 The following graphic illustrates this by using the Run element embedded in a Span element:
```

File: docs/framework/wpf/advanced/bidirectional-features-in-wpf-overview.md [hunk 3]
```diff
@@ -57,17 +53,13 @@ Unlike any other development platform, [!INCLUDE[TLA2#tla_winclient](../../../..
...
   
  Because the text is <xref:System.Windows.FlowDirection.RightToLeft>, all special characters, such as the "\\", separate the text in a right to left order. That results in the link not being shown in the correct order, therefore to solve the problem, the text must be embedded to preserve a separate <xref:System.Windows.Documents.Run> flowing <xref:System.Windows.FlowDirection.LeftToRight>. Instead of having a separate <xref:System.Windows.Documents.Run> for each language, a better way to solve the problem is to embed the less frequently used English text into a larger Arabic <xref:System.Windows.Documents.Span>.  
   
- The following graphic illustrates this.  
-  
- **Graphic That Illustrates Using the Run Element Embedded in a Span Element**  
+ The following graphic illustrates this.   
   
- ![XamlPad screen shot](./media/runspan.PNG "RunSpan")  
+ ![Graphic illustrates the Run element embedded in a Span element.](./media/bidirectional-features-in-wpf-overview/embedded-span-element.png)
```
Comment (mairaw) [inline docs/framework/wpf/advanced/bidirectional-features-in-wpf-overview.md]:
```suggestion
 ![Graphic that illustrates the Run element embedded in a Span element.](./media/bidirectional-features-in-wpf-overview/embedded-span-element.png)  
```

File: docs/framework/wpf/advanced/bidirectional-features-in-wpf-overview.md [hunk 4]
```diff
@@ -115,18 +101,14 @@ Unlike any other development platform, [!INCLUDE[TLA2#tla_winclient](../../../..
 <a name="FlowDirectionNontext"></a>   
 ## FlowDirection with Non-text Elements  
  <xref:System.Windows.FlowDirection> defines not only how text flows in a textual element but also the flow direction of almost every other [!INCLUDE[TLA2#tla_ui](../../../../includes/tla2sharptla-ui-md.md)] element. The following graphic shows a <xref:System.Windows.Controls.ToolBar> that uses a horizontal <xref:System.Windows.Media.LinearGradientBrush> to draw its background.
```
Comment (mairaw) [inline docs/framework/wpf/advanced/bidirectional-features-in-wpf-overview.md]:
```suggestion

 <xref:System.Windows.FlowDirection> defines not only how text flows in a textual element but also the flow direction of almost every other [!INCLUDE[TLA2#tla_ui](../../../../includes/tla2sharptla-ui-md.md)] element. The following graphic shows a <xref:System.Windows.Controls.ToolBar> that uses a horizontal <xref:System.Windows.Media.LinearGradientBrush> to draw its background with a left to right gradient.  
```

File: docs/framework/wpf/advanced/bidirectional-features-in-wpf-overview.md [hunk 5]
```diff
@@ -164,9 +144,7 @@ Unlike any other development platform, [!INCLUDE[TLA2#tla_winclient](../../../..
   
  The following graphic shows the output of the previous example.
```
Comment (mairaw) [inline docs/framework/wpf/advanced/bidirectional-features-in-wpf-overview.md]:
```suggestion
 The following graphic shows the output of the previous example with arrows drawn using the `Path` element:
```

File: docs/framework/wpf/advanced/bidirectional-features-in-wpf-overview.md [hunk 6]
```diff
@@ -164,9 +144,7 @@ Unlike any other development platform, [!INCLUDE[TLA2#tla_winclient](../../../..
   
  The following graphic shows the output of the previous example.  
   
- **Graphic That Illustrates Arrows Drawn Using the Path Element**  
-  
- ![Paths](./media/paths.PNG "Paths")  
+ ![Graphic that illustrates arrows drawn using the path element.](./media/bidirectional-features-in-wpf-overview/arrows-drawn-path-element.png)
```
Comment (mairaw) [inline docs/framework/wpf/advanced/bidirectional-features-in-wpf-overview.md]:
```suggestion
 ![Graphic that illustrates arrows drawn using the Path element.](./media/bidirectional-features-in-wpf-overview/arrows-drawn-path-element.png)  
```

File: docs/framework/wpf/advanced/bidirectional-features-in-wpf-overview.md [hunk 7]
```diff
@@ -213,10 +191,8 @@ Unlike any other development platform, [!INCLUDE[TLA2#tla_winclient](../../../..
  [!code-xaml[Numbers#Numbers](~/samples/snippets/csharp/VS_Snippets_Wpf/Numbers/CS/Window1.xaml#numbers)]  
   
  The following graphic shows the output of the previous sample if you are running in an Arabic version of [!INCLUDE[TLA#tla_mswin](../../../../includes/tlasharptla-mswin-md.md)].
```
Comment (mairaw) [inline docs/framework/wpf/advanced/bidirectional-features-in-wpf-overview.md]:
```suggestion
 The following graphic shows the output of the previous sample if you're running in an Arabic version of Windows with Arabic and English numbers displayed:
```

File: docs/framework/wpf/advanced/bidirectional-features-in-wpf-overview.md [hunk 8]
```diff
@@ -260,10 +236,8 @@ text1.Language = System.Windows.Markup.XmlLanguage.GetLanguage(System.Globalizat
  [!code-csharp[NumbersCSharp#NumbersCSharp](~/samples/snippets/csharp/VS_Snippets_Wpf/NumbersCSharp/CSharp/Window1.xaml.cs#numberscsharp)]  
   
  The following graphic shows what the window looks like for either programming language.
```
Comment (mairaw) [inline docs/framework/wpf/advanced/bidirectional-features-in-wpf-overview.md]:
```suggestion
 The following graphic shows what the window looks like for either programming language, displaying Arabic numbers:
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Changes look great @damabe. Left a few suggestions that can be batched in a single via the [Files changed](https://github.com/dotnet/docs/pull/11249/files) tab, Some changes will have to be manual since they require to rename the image name.

**Comment by damabe** [comment]
@mairaw Fixes have been made. CATS report: https://contentqacats.azurewebsites.net/CheckResults/2019031300037/Report/

**Review by mairaw — APPROVED** [review]
Thank you @damabe!

**Pull request merged**